presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -68,6 +68,7 @@
     public static final String LEGACY_ORDER_BY = "legacy_order_by";
     public static final String REORDER_WINDOWS = "reorder_windows";
     public static final String ITERATIVE_OPTIMIZER = "iterative_optimizer_enabled";
+    public static final String AGGREGATION_PUSHDOWN = "aggregation_pushdown_enabled";
```

Comment:
please consider naming it consistently with already existing property:

`PUSH_TABLE_WRITE_THROUGH_UNION = "push_table_write_through_union";`

Like, `push_aggregation_through_join`.